Refactors "strpos" calls in  lib/private to improve code readability.

fsamapoor opened the pull request:
- [x] Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [x] [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits

Commit:
Refactors "strpos" calls in  lib/private to improve code readability.

Signed-off-by: Faraz Samapoor <fsamapoor@gmail.com>

nfebe reviewed (approved):
LGTM

icewind1991 commented on lib/private/Files/Type/Detection.php:
```diff
@@ -238,15 +238,15 @@ public function detectContent(string $path): string {
 		}
 
-		if (strpos($path, '://') !== false && strpos($path, 'file://') === 0) {
+		if (str_contains($path, '://') && str_starts_with($path, 'file://')) {
```
these 2 can be combined as the latter implies the former

icewind1991 commented on lib/private/TemplateLayout.php:
```diff
@@ -284,7 +284,7 @@ public function __construct($renderAs, $appId = '') {
 				$suffix = $this->getVersionHashSuffix($web, $file);
 
-				if (strpos($file, '?v=') == false) {
+				if (!strpos($file, '?v=')) {
```
`!str_contains`

fsamapoor replied:
Thanks. Done.

fsamapoor committed:
Update Detection.php

The latter implies the former.

https://github.com/nextcloud/server/pull/38261#discussion_r1196708412

Signed-off-by: Faraz Samapoor <f.samapoor@gmail.com>

fsamapoor committed:
Update TemplateLayout.php

https://github.com/nextcloud/server/pull/38261#discussion_r1196711637

Signed-off-by: Faraz Samapoor <f.samapoor@gmail.com>
